core/iwasm/common/wasm_runtime_common.c

```diff
@@ -30,6 +30,9 @@
 #if WASM_ENABLE_FAST_JIT != 0
 #include "../fast-jit/jit_compiler.h"
 #endif
+#if WASM_ENABLE_JIT != 0 || WASM_ENABLE_WAMR_COMPILER != 0
```

Comment:
just curious, not an opinion. Can we use `#if WASM_ENABLE_AOT != 0` here?

Reply:
No, macro WASM_ENABLE_WAMR_COMPILER is defined by wamrc CMakeLists.txt, and is wildly used in wasm_loader.c and some other places.

Theoretically, LLVM JIT and wamrc don't definitely depend on AOT, using `WASM_ENABLE_AOT` is not a good choice. And enabling AOT doesn't mean that LLVM JIT and wamrc are enabled, using `WASM_ENABLE_AOT` will cause to compile error for iwasm when JIT isn't built in.